samples/meeting-av-filters/csharp/MeetingAVFilter/AppPackage/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.7/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
+  "version": "1.0.0",
+  "id": "<<GUID ID>>",
+  "packageName": "com.microsoft.teams.videopp",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://microsoft.github.io/teams-videoapp-sample/",
```

Comment:
Do we have this URL exist ?

Reply:
Updated it to be inline with other samples in the repo